README.md

```diff
@@ -57,6 +57,15 @@ Features that we do not currently support, but we do plan to add in the future:
 - cd to `example` folder
 - run `flutter pub get` to install dependencies
 - `flutter run` will start the application on connected android / iOS device
+- Note for M1 Macs:
```

Comment:
I would suggest starting a new section here:

```md
## Troubleshooting the example
```

Because hopefully most users will not hit these issues, so that keeps "Running the example" as pure "golden path" instructions.

Reply:
I've added this and also added `## Example App` and `### Running example app`. 9764e94